modules/websearch/lib/websearch_templates.py

```diff
@@ -140,7 +140,7 @@ class Template:
         'sf': (str, ""),
         'so': (str, "d"),
         'sp': (str, ""),
-        'rm': (str, ""),
+        'rm': (str, "recommendations"),
```

Comment:
Needs to be "" by default.  You can use `CFG_* variable` to  decide to set to it something else, for those sites that enabled Obelix.